File: Data Structures & Algorithms/longest-consecutive-sequence/submission-0.cpp
class Solution {
public:
    int longestConsecutive(vector<int>& nums) {
        unordered_set<int> numSet;
        int longest = 0;
        for(int num : nums)
            numSet.insert(num);

        for(auto num : numSet)
        {
            if( numSet.count(num-1) == 0 )
            {
                int length = 1;
                while( numSet.count(num + length) > 0)
                {
                    length += 1;
                }
                longest = max(length, longest);
            }
        }
        return longest;
    }
};

File: Data Structures & Algorithms/buy-and-sell-crypto/submission-0.py
class Solution:
    def maxProfit(self, prices: List[int]) -> int:
        res = 0
        i = 0
        j = i
        
        while j < len(prices):
            res = max(res, prices[j] - prices[i])

            if prices[j] <= prices[i]:
                i = j
            
            j += 1

        return res
        